code/modules/martial_arts/martial.dm

```diff
@@ -289,6 +289,24 @@
 	new /obj/effect/decal/cleanable/ash(get_turf(src))
 	qdel(src)
 
+/obj/item/adv_first_aid_manual
+	name = "Advanced First Aid Manual"
+	desc = "A suspicious manual with medical iconography"
+	icon = 'icons/obj/library.dmi'
+	icon_state = "adv_first_aid_manual"
+
+/obj/item/adv_first_aid_manual/attack_self(mob/living/carbon/human/user)
+	if(!istype(user))
+		return
+	to_chat(user, "<span class='boldannounce'>You flip through the pages quickly and pick up on a couple of ways to treat serious injuries.</span>")
```

Comment:
You could probably have something like this on the `teach` proc and likewise on the `remove` proc. so when they learn the martial art it says "You have learnt advanced first aid! You are now  extremely skilled in fixing internal bleeding and broken bones." and when the martial art is removed it says something like "You have forgotten advanced first aid!". just makes it more clear to users what is happening. you can take a look at the code in `krav_maga.dm` for examples